fenicsprecice/adapter_core.py

```diff
@@ -409,7 +410,7 @@ def edge_is_on(subdomain, edge):
     return vertices1_ids, vertices2_ids
 
 
-def get_forces_as_point_sources(fixed_boundary, function_space, data):
```

Comment:
Is `dims` really needed? I think this should already be known by the adapter from initialization.

Reply:
After a discussion with @BenjaminRodenberg it was concluded that a major restructuring in `adapter_core.py` is not necessary at this point.